src/Uno.UWP/Storage/StorageFile.wasm.cs

```diff
@@ -24,6 +24,11 @@ private static async Task<StorageFile> GetFileFromApplicationUri(CancellationTok
 
 			var path = Uri.UnescapeDataString(uri.PathAndQuery);
 
+			if(uri.Host is { Length: > 0 } host)
+			{
+				path = host + "/" + path;
```

Comment:
nit: Path.Combine here?

Reply:
`Path.Combine` is about filesystem paths, in here it's an actual URI, so it needs to be a `/`. Still, because it's wasm so it would work, but just because emscripten FS is unix based.